code/modules/admin/verbs/adminhelp.dm

```diff
@@ -91,24 +92,50 @@
 	if(src.handle_spam_prevention(msg,MUTE_ADMINHELP))
 		return
 
+	var/ref_client = "\ref[src]"
+	for(var/datum/adminticket/T in admintickets)
+		if(T.permckey == src.ckey && T.resolved == "No")
+			if(alert(usr,"You already have an adminhelp open, would you like to bump it?", "Bump Adminhelp", "Yes", "No") == "Yes")
+				T.logs += "[src.ckey] has bumped this adminhelp!"
+				if(T.admin == "N/A")
+					usr << "<b>Due to the fact your Adminhelp had no assigned admin, admins have been pinged.</b>"
+					message_admins("[src.ckey] has bumped their adminhelp #[T.ID], still no assigned admin!")
+					msg = "<span class='adminnotice'><b><font color=red>HELP: </font><A HREF='?priv_msg=[ckey];ahelp_reply=1'>[key_name(src)]</A> [ADMIN_QUE(mob)] [ADMIN_PP(mob)] [ADMIN_VV(mob)] [ADMIN_SM(mob)] [ADMIN_FLW(mob)] [ADMIN_TP(mob)] (<A HREF='?_src_=holder;rejectadminhelp=[ref_client]'>REJT</A>) (<A HREF='?_src_=holder;icissue=[ref_client]'>IC</A>) (<A HREF='?_src_=ticket;resolve=[T.ID]'>R</a>):</b> [msg]</span>"
+					for(var/client/X in admins)

```

Comment:
Please use _istypeless for loops_ you can find more information about this in the [contributing.md](https://github.com/HippieStation/HippieStation13/blob/master/CONTRIBUTING.md)